src/main/webapp/app/pages/AccountPage.tsx

```diff
@@ -212,37 +224,82 @@ export class AccountPage extends React.Component<IRegisterProps> {
                 className={'ml-2'}
               />
             </div>
-            <InfoRow
-              title={
-                <div className={'d-flex align-items-center'}>
-                  <span>
-                    {getAccountInfoTitle(
-                      ACCOUNT_TITLES.API_TOKEN,
-                      this.account.licenseType as LicenseType
-                    )}
-                  </span>
-                  {this.generateTokenEnabled ? (
-                    <DefaultTooltip
-                      placement={'top'}
-                      overlay={'Get a new token.'}
-                    >
-                      <i
-                        className={classnames('ml-2 fa fa-plus')}
-                        onClick={this.addNewToken}
-                      />
-                    </DefaultTooltip>
-                  ) : null}
-                </div>
-              }
-            >
-              <TokenInputGroups
-                changeTokenExpirationDate={false}
-                tokens={this.tokens}
-                onDeleteToken={this.deleteToken}
-              />
-            </InfoRow>
+            {this.account.authorities.includes(AUTHORITIES.API) ? (
```

Comment:
We should also handle the case when user has requested the API access. A language like  "We are reviewing your API access request. If you haven't heard back from us after 5 business days, please feel free to contact us." should work.